### ISSUE TITLE
[Example]: ERC721 example

### ISSUE THREAD

**Issue by rsodre** [dojoengine/origami#73]
### Feature Request

Create a simple Dojo project deploying a new ERC721 token.

### Proposed Solution

* Dojo project deploying a new ERC721 token.
* React project able to connect a wallet, mint tokens and display metadata.

#### TBD:
* What's the best way to structure this example?
  * Dojo project at [dojo/examples](https://github.com/dojoengine/dojo/tree/main/examples)?
  * React dapp at [dojo.js/examples](https://github.com/dojoengine/dojo.js/tree/main/examples)?
  * Both under the same place? Where?

### If the feature is accepted, would you be willing to contribute it?

- [X] Yes I would be willing to contribute

**Comment by Ejembiii** [comment]
@rsodre I would like to work on this issue if it is not assigned to anyone

**Comment by karlostoteles** [comment]
Hey @ooochoche and @rsodre what's the state of this issue?

**Comment by rsodre** [comment]
> Hey @ooochoche and @rsodre what's the state of this issue?

I made a side project that could be used as a base for the example [here](https://github.com/rsodre/512karat).
But if you try to build it, it will probably fail, as my origami branch was upgraded to Dojo 1.0, and this project is not yet.

Also, my origami branch has some updates proposed as #79  and #81, which I don't know if will be merged or not.

**Comment by rsodre** [comment]
> @rsodre I would like to work on this issue if it is not assigned to anyone

Hey, I'm not in charge here, just proposing...

**Comment by karlostoteles** [comment]
> as

Yeah I already looked on that project you sent me on discord. In this case with all this versioning issue what would you then? Working on alpha.3? I think Glihm updated that for origami purposes (?). And thank you!

**Comment by rsodre** [comment]
I updated mine to alpha-4

**Comment by karlostoteles** [comment]
but the latest origami branch is at alpha-3 I think right?

**Comment by rsodre** [comment]
@karlostoteles you can use this:
https://github.com/rsodre/origami/tree/dojo_1.0.0-alpha.4
